package.json

```diff
@@ -16,16 +16,19 @@
   "scripts": {
     "dev": "vite build --watch & yarn storybook",
     "test": "jest",
+    "test-ci": "jest --collectCoverage --coverageDirectory=build-reports/coverage",
```

Comment:
Let's add a `jest.config.ci` file that extends the normal one with the ci specific config?

Reply:
I would defer until we need to override enough that makes the script too long 🤷🏻